src/common/config_opts.h

```diff
@@ -1134,6 +1134,7 @@ OPTION(filestore_kill_at, OPT_INT, 0)            // inject a failure at the n'th
 OPTION(filestore_inject_stall, OPT_INT, 0)       // artificially stall for N seconds in op queue thread
 OPTION(filestore_fail_eio, OPT_BOOL, true)       // fail/crash on EIO
 OPTION(filestore_debug_verify_split, OPT_BOOL, false)
+OPTION(filestore_pglog_columnfamily, OPT_BOOL, false)
```

Comment:
Have you missed mentioning tracker of backport? Sorry if i missed.
How to verify backport PR?